www/heatmap.html

```diff
@@ -0,0 +1,239 @@
+            </div>
+            <hr class="featurette-divider">
+           <a href="https://www.patrick-wied.at/static/heatmapjs/">Heatmap.js</a>, created by <a href="https://www.patrick-wied.at">Patrick Wied,</a> is a "lightweight, easy to use JavaScript library to help you visualize your three dimensional data!" <a href="https://www.patrick-wied.at/static/heatmapjs/get-heatmap.html">Download</a> the library and save the <code class="javascript">heatmap.js</code> file into the same directory as your site's javascript files. View the <a href="https://www.patrick-wied.at/static/heatmapjs/docs.html">documentation</a> and then follow this quick tutorial to implement the library with your WebGazer data.
+        </div>
+
+        <div id="video" class="container pad">
+            <hr class="featurette-divider">
+            <h2 class="text-center" style="margin-bottom:0px">Live Demo</h2><br><p class="text-center padright" style="font-size:20px">This demo does not work with an ad-blocker.</p>
+            <div id="heatmapContainer" style="margin: auto; width: 100%; height: auto; max-width: 700px">
+                    <img id="webgazer-pic" src="./media/heatmap-demo.png" style="width: 100%; height: auto; max-width: 700px">
+            </div>
+            <script src="./js/heatmap.js"></script>
+            <script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
+            <script src="./js/heatmap-data.js"></script>
+            <script src="./js/heatmap-webgazer.js"></script>
+       </div>
+
+        <div id="heatmap-usage" class="container pad">
+            <hr class="featurette-divider">
+            <h2 class="text-center">Usage</h2>
+            <br>Begin by creating a div element that will house all of the heatmap's visual rendering on your page.
+            
+<pre><code class="html">&lt;!-- Heatmap Container --&gt;
+&lt;div id="heatmapContainer">&lt;div&gt;</code></pre>
+
+            <p><br>Create HTML script elements that import the javascript files needed for the visualization. The script element needs to be given the location that the <code class="javascript">heatmap.js</code> file is stored. <b>Note:</b> The period in <code class="javascript">"./js/heatmap.js"</code> represents the current directory.</p>
+
+<pre><code class="html">&lt;!-- Script Element --&gt;
+&lt;script src="./js/heatmap.js"&gt;&lt;script&gt; 
+</code></pre>
+
+<p><br>Initialize a heatmap instance with a few lines of javascript.</p>
+<pre><code class="javascript hljs">var heatmap = h337.create({
+    container: document.getElementById('heatmapContainer') // refers to the container that you created above
+});
+</code></pre>
+
+            <p><code class="javascript">Heatmap.js</code> supports many kinds of visual configurations that change the heatmaps appearance. View these settings in the initialization <a href="https://www.patrick-wied.at">documentation.</a><p>
```

Comment:
Should this link instead to the heatmap documentation at https://www.patrick-wied.at/static/heatmapjs/docs.html